<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Help</class>
 <widget class="QWidget" name="Help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vHelpLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="tBHome">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
        <property name="icon">
         <iconset resource="help.qrc">
          <normaloff>:/whelp/go-home.png</normaloff>:/whelp/go-home.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>33</width>
          <height>33</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tBPrevious">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;--</string>
        </property>
        <property name="icon">
         <iconset resource="help.qrc">
          <normaloff>:/whelp/go-previous.png</normaloff>:/whelp/go-previous.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>33</width>
          <height>33</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tBNext">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>--&gt;</string>
        </property>
        <property name="icon">
         <iconset resource="help.qrc">
          <normaloff>:/whelp/go-next.png</normaloff>:/whelp/go-next.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>33</width>
          <height>33</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEUrl">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="dummy" native="true"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="help.qrc"/>
 </resources>
 <connections/>
</ui>
